### ISSUE TITLE
_Float128 is unrecognized

### ISSUE THREAD

**Issue by jchia** [visq/language-c#39]
Under gcc 7.2 with glibc 2.26 on Arch Linux, parsing `/usr/include/stdlib.h` with the option `-D__STDC_WANT_IEC_60559_TYPES_EXT__` results in an error:
```
  Syntax error !
  The symbol `strtof128' does not fit here.
```

The offending code from `stdlib.h` is as follows:
```
#if __HAVE_FLOAT128 && __GLIBC_USE (IEC_60559_TYPES_EXT)
/* Likewise for the '_Float128' format  */
extern _Float128 strtof128 (const char *__restrict __nptr,
                      char **__restrict __endptr)
     __THROW __nonnull ((1));
#endif
```

To repro, take the `BasicUsage.hs` example and:
- parse "/usr/include/stdlib.h" instead of "test.c"
- use `parseCFile` options `["-D__STDC_WANT_IEC_60559_TYPES_EXT__"]` instead of `[]`
- run the example on a problematic platform (current Arch Linux should suffice, but I think another system with with gcc 7.2 and glibc 2.26 should also suffice)

Relevant link about `_Float128`: https://gcc.gnu.org/onlinedocs/gcc-7.2.0/gcc/Floating-Types.html#Floating-Types

**Comment by jchia** [comment]
Is the correct fix to make `language-c` recognize `_Float128` like it already does `__float128` (#33)?

I'm not clear why gcc has both `__float128` and `_Float128`. I only see that they exist.

**Comment by visq** [comment]
Fixed by #41
